.github/workflows/ci.yml

```diff
@@ -0,0 +1,76 @@
+name: Build
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+env:
+  GO111MODULE: on
+  DEBIAN_FRONTEND: noninteractive
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: ['ubuntu-18.04', 'macOS-10.15', 'windows-2019']
+        go: ['1.14.x']
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go }}
+    - run: |
+        make gofmt lint
+        go test -race -covermode atomic -coverprofile=profile.cov ./...
+      shell: bash
+    - uses: shogo82148/actions-goveralls@v1
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        path-to-profile: profile.cov
+        parallel: true
+        job-number: ${{ strategy.job-index }}
+  finish:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: shogo82148/actions-goveralls@v1
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        parallel-finished: true
+  release:
+    needs: test
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+          rpm devscripts debhelper fakeroot \
+          crossbuild-essential-arm64 crossbuild-essential-armhf
+        mkdir -p ~/rpmbuild/{BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
+    - uses: actions/setup-go@v2
```

Comment:
Should we also use `actions/cache@v2` for `~/go/pkg/mod` here?

Reply:
I overlooked that; will add it soon.